server/game/cards/01_SOR/upgrades/SnapshotReflexes.ts

```diff
@@ -0,0 +1,23 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { UpgradeCard } from '../../../core/card/UpgradeCard';
+
+export default class SnapshotReflexes extends UpgradeCard {
+    protected override getImplementationId() {
+        return {
+            id: '9985638644',
+            internalName: 'snapshot-reflexes'
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.addWhenPlayedAbility({
+            title: 'You may attack with attached unit.',
```

Comment:
nit: for ability titles, we don't include "You may" since there will be "pass" button making it clear that the effect is optional. Also we shouldn't have a period at the end (we haven't been consistent about that, I need to go through and clean up at some point)

Reply:
done, thanks for clarification